.devcontainer/setup.sh

```diff
@@ -0,0 +1,4 @@
+poetry install
+poetry run pytest
+poetry run mypy
+poetry run pre-commit install
```

Comment:
We should drop these lines; if the project is in a failing state we shouldn't cause issues with Dev Container creation... Imagine if a user modifies some files such that ruff is not happy, and then tries to activate Dev Containers with their editor -- now they have to stash or similar to avoid a failure here (and not all users will understand that).